components/eamxx/scripts/test_all_scream.py

```diff
@@ -831,14 +716,11 @@ def baselines_to_be_generated(self):
         """
         Return list of baselines to generate. Baselines need to be generated if
          - they are missing
-         - they are expired and we asked to update expired baselines
```

Comment:
How do we force new baselines now that you removed `--force-baseline-regen`? I think we should either
- keep `--force-baseline-regen`, which generates baselines even if not missing
- always generate baselines, even if not missing.

I think the former may be more robust against accidental typos (like `-g` instead of `-h`). But the latter reduces the code in TAS... I'm ok either way, but I would like to have to manually rm baselines if I want them regenerated...

Reply:
Good catch, this was a mistake.